lib/modules/redis/redis.go

```diff
@@ -0,0 +1,33 @@
+package redis
+
+import (
+	"os"
+	"strconv"
+
+	"github.com/go-redis/redis"
+)
+
+type Redis struct {
+	Client *redis.Client
+}
+
+var sharedInstance = New()
+
+func New() *Redis {
```

Comment:
__[Golint]__ exported function New should have comment or be unexported ([link](https://sider.review/gh/repos/50192141/pulls/456#issue-3827452))

<sub>posted by Sider</sub>